.scalafmt.conf

```diff
@@ -1,25 +1,10 @@
-style = IntelliJ,
+version = "2.3.1"
```

Comment:
We could already update to the latest version: https://github.com/scalameta/scalafmt/releases